applications/plugins/SofaPython/Binding_Data.cpp

```diff
@@ -722,8 +722,10 @@ extern "C" PyObject * Data_getValueVoidPtr(PyObject * self, PyObject * /*args*/)
     return res;
 }
 
-
-
+extern "C" PyObject * Data_getAsACreateObjectParameter(PyObject * self, PyObject * args)
```

Comment:
You are right, 
But as I hate inconsistencies... having static somewhere and extern two line later hurts me...
because this is the kind of things that will stay for a very very long time. 

I would be much more in favore of a single PR in which all those are replaced in a batch.